tools/reactive-controllers/element-resolution.md

```diff
@@ -14,69 +34,362 @@ yarn add @spectrum-web-components/reactive-controllers
+import { ElementResolutionController } from '@spectrum-web-components/reactive-controllers/src/ElementResolution.js';
+
+class FormController extends LitElement {
+    resolvedElement = new ElementResolutionController(this, {
+        selector: '#submit-button',
+    });
+
+    handleSubmit() {
+        if (this.resolvedElement.element) {
+            this.resolvedElement.element.click();
+        }
+    }
+}
+
+customElements.define('form-controller', FormController);
+```
+
+#### Tracking resolution updates
 
 Changes to the resolved element reference are reported to the host element via a call to the `requestUpdate()` method. This will be provided the `elementResolverUpdatedSymbol` as the changed key. If your element leverages this value against the changes map, it can react directly to changes in the resolved element:
 
-```ts
-import { html, LitElement } from 'lit';
+```typescript
+import { html, LitElement, PropertyValues } from 'lit';
 import {
     ElementResolutionController,
     elementResolverUpdatedSymbol,
-} from '@spectrum-web-components/reactive-controllers/ElementResolution.js';
+} from '@spectrum-web-components/reactive-controllers/src/ElementResolution.js';
 
 class RootEl extends LitElement {
     resolvedElement = new ElementResolutionController(this);
 
-    costructor() {
+    constructor() {
         super();
         this.resolvedElement.selector = '.other-element';
     }
 
     protected override willUpdate(changes: PropertyValues): void {
         if (changes.has(elementResolverUpdatedSymbol)) {
-            // work to be done only when the element reference has been updated
+            // Work to be done only when the element reference has been updated
+            console.log(
+                'Resolved element changed:',
+                this.resolvedElement.element
+            );
+        }
+    }
+
+    render() {
+        return html`
+            <p>
+                Element status:
+                ${this.resolvedElement.element ? 'Found' : 'Not found'}
+            </p>
+        `;
+    }
+}
+
+customElements.define('root-el', RootEl);
+```
+
+#### Accessible label resolution
+
+Use `ElementResolutionController` to resolve accessible labeling elements:
+
+```typescript
+import { html, LitElement } from 'lit';
+import { ElementResolutionController } from '@spectrum-web-components/reactive-controllers/src/ElementResolution.js';
+
+class CustomInput extends LitElement {
+    labelElement = new ElementResolutionController(this, {
+        selector: '.input-label',
+    });
+
+    firstUpdated() {
+        // Connect input to label for accessibility
+        if (this.labelElement.element) {
+            const labelId = this.labelElement.element.id || this.generateId();
+            this.labelElement.element.id = labelId;
+
+            const input = this.shadowRoot?.querySelector('input');
+            if (input) {
+                input.setAttribute('aria-labelledby', labelId);
+            }
+        }
+    }
```

Comment:
I don't think this part really exemplifies the real reason someone would want to use this, which is as a cross-root ARIA solution. I'd suggest including some of the logic from field label:

```
if (targetParent === (this.getRootNode() as HTMLElement)) {
                const conditionAttribute = target
                    ? conditionAttributeWithId
                    : conditionAttributeWithoutId;
                conditionAttribute(focusable, 'aria-labelledby', [this.id]);
            } else {
                if (target) {
                    focusable.setAttribute('aria-label', this.labelText);
                } else {
                    focusable.removeAttribute('aria-label');
                }
            }
```